lab-doug/gulpfile.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+var gulp = require('gulp');
+var eslint = require('gulp-eslint');
+var mocha = require('gulp-mocha');
+
+gulp.task('default', ['mocha-test', 'eslint-test']
+);
```

Comment:
Great work!!  Only thing I'd change is moving line 8 to line 7, one line tasks are fine.